src/server/scripts/KulTiras/ShrineOfTheStorm/boss_tidesage_council.cpp

```diff
@@ -0,0 +1,541 @@
+    {
+        return ValidateSpellInfo({ SPELL_BLOWBACK_MISSILE });
+    }
+
+    void HandleProc(ProcEventInfo const& /*eventInfo*/) const
+    {
+        GetTarget()->CastSpell(GetTarget(), SPELL_BLOWBACK_MISSILE, TRIGGERED_IGNORE_CAST_IN_PROGRESS | TRIGGERED_DONT_REPORT_CAST_ERROR);
+    }
+
+    void Register() override
+    {
+        OnProc += AuraProcFn(spell_tidesage_council_blessing_of_the_tempest::HandleProc);
+    }
+};
+
+// 267905 - Reinforcing Ward
+struct at_tidesage_council_reinforcing_ward : AreaTriggerAI
+{
+    using AreaTriggerAI::AreaTriggerAI;
+
+    void OnUnitEnter(Unit* unit) override
+    {
+        if (unit->GetEntry() == BOSS_BROTHER_IRONHULL || unit->GetEntry() == BOSS_GALECALLER_FAYE)
+        {
+            unit->CastSpell(unit, SPELL_REINFORCING_WARD_DAMAGE_TAKEN_BOSS, TRIGGERED_IGNORE_CAST_IN_PROGRESS | TRIGGERED_DONT_REPORT_CAST_ERROR);
+            unit->CastSpell(unit, SPELL_REINFORCING_WARD_DISPEL_BOSS, TRIGGERED_IGNORE_CAST_IN_PROGRESS | TRIGGERED_DONT_REPORT_CAST_ERROR);
+        }
+        else
+        {
+            unit->CastSpell(unit, SPELL_REINFORCING_WARD_DAMAGE_TAKEN_PLAYER, TRIGGERED_IGNORE_CAST_IN_PROGRESS | TRIGGERED_DONT_REPORT_CAST_ERROR);
+            unit->CastSpell(unit, SPELL_REINFORCING_WARD_DISPEL_PLAYER, TRIGGERED_IGNORE_CAST_IN_PROGRESS | TRIGGERED_DONT_REPORT_CAST_ERROR);
+        }
+    }
+
+    void OnUnitExit(Unit* unit) override
+    {
+        unit->RemoveAurasDueToSpell(SPELL_REINFORCING_WARD_DAMAGE_TAKEN_BOSS);
+        unit->RemoveAurasDueToSpell(SPELL_REINFORCING_WARD_DAMAGE_TAKEN_PLAYER);
+    }
+
+    void OnCreate(Spell const* /*creatingSpell*/) override
+    {
+        _scheduler.Schedule(1s, [this](TaskContext task)
+        {
+            for (ObjectGuid const& guid : at->GetInsideUnits())
+            {
+                Unit* unit = ObjectAccessor::GetUnit(*at, guid);
+                if (!unit)
+                    continue;
+
+                if (unit->isDead())
+                    continue;
+
+                if (unit->GetEntry() == BOSS_BROTHER_IRONHULL || unit->GetEntry() == BOSS_GALECALLER_FAYE)
+                    unit->CastSpell(unit, SPELL_REINFORCING_WARD_DISPEL_BOSS, TRIGGERED_IGNORE_CAST_IN_PROGRESS | TRIGGERED_DONT_REPORT_CAST_ERROR);
+                else
+                    unit->CastSpell(unit, SPELL_REINFORCING_WARD_DISPEL_PLAYER, TRIGGERED_IGNORE_CAST_IN_PROGRESS | TRIGGERED_DONT_REPORT_CAST_ERROR);
+            }
+            task.Repeat(1s);
+        });
+    }
+
+    void OnUpdate(uint32 diff) override
+    {
+        _scheduler.Update(diff);
+    }
+
+private:
+    TaskScheduler _scheduler;
+};
+
+// 267891 - Swiftness Ward
+// Id - 13154
+struct at_tidesage_council_swiftness_ward_boss : AreaTriggerAI
+{
+    using AreaTriggerAI::AreaTriggerAI;
+
+    void OnUnitEnter(Unit* unit) override
+    {
+        if (unit->GetEntry() != BOSS_BROTHER_IRONHULL || unit->GetEntry() != BOSS_GALECALLER_FAYE)
```

Comment:
wrong condition, replace with:


```suggestion
if (unit->GetEntry() != BOSS_BROTHER_IRONHULL && unit->GetEntry() != BOSS_GALECALLER_FAYE)
    return;
```